components/resources/quizzes/langQuiz/LangResults.tsx

```diff
@@ -3,22 +3,70 @@ import SkillifyNavbar from "../shared/SkillifyNavbar";
 
 type LangResultsProps = {
   onBackClick: () => void;
+  score: { [key: string]: number };
```

Comment:
Use the typescript Record  type and type aliases for this. 

```
type LanguageQuizResultsMap = Record<string, number>;
```

```
score: LanguageQuizResultsMap
```

https://www.typescriptlang.org/docs/handbook/advanced-types.html#type-aliases
https://stackoverflow.com/questions/51936369/what-is-the-record-type